sky/skylet/providers/aws/config.py

```diff
@@ -33,20 +33,21 @@
 DEFAULT_RAY_IAM_ROLE = RAY + "-v1"
 SECURITY_GROUP_TEMPLATE = RAY + "-{}"
 
-DEFAULT_AMI_NAME = "AWS Deep Learning AMI (Ubuntu 18.04) V30.0"
+# V61.0 has CUDA 11.2
```

Comment:
Q: do we want to update other clouds under `sky/skylet/providers/` to 1.13? In skylet's LICENSE:
```
Code in providers/azure from
https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/_private/_azure
Git Revision: 7f1bacc7dc9caf6d0ec042e39499bbf1d9a7d065

Code in providers/gcp from
https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/_private/gcp
Git Revision: ef9d9df4e7454d428a958281e9de333795dccb44

Code in providers/aws from
https://github.com/ray-project/ray/tree/master/python/ray/autoscaler/_private/aws
Git Revision: f737731a5e53ab3a2c5192d3d9efd4bb4bbbbf48
```

Reply:
I meant that those code were forked from a pre-0.13 version. Since we're supposed to upgrade Ray to 0.13 completely, should we upgrade those forked files, too?